resdk/tests/functional/base.py

```diff
@@ -87,7 +96,7 @@ def get_reads(self, count=1, collection=None):
             collections.
         :type collection: None, int or `~resdk.resources.Collection`
         """
-        reads_path = os.path.join(FILES_PATH, 'dummy_reads.fastq')
```

Comment:
Why have you renamed this?

Reply:
`dummy_reads.fastq` was returning errors upon attempting upload, but reads.fastq.gz seems to work. Should this be a separate commit?